FlowCal/mef.py

```diff
@@ -265,27 +320,34 @@ def fit_beads_autofluorescence(fl_channel, fl_mef):
     # Initial guesses:
     # 0: slope found by putting a line through the highest two points.
     # 1: y-intercept found by putting a line through highest two points.
-    # 2: bead autofluorescence initialized to 100.
+    # 2: bead autofluorescence initialized using the first point.
     params[0] = (np.log(fl_mef[-1]) - np.log(fl_mef[-2])) / \
-                    (fl_channel[-1] - fl_channel[-2])
-    params[1] = np.log(fl_mef[-1]) - params[0] * fl_channel[-1]
-    params[2] = 100.
+                (np.log(fl_au[-1]) - np.log(fl_au[-2]))
+    params[1] = np.log(fl_mef[-1]) - params[0] * np.log(fl_au[-1])
+    params[2] = np.exp(params[0]*np.log(fl_au[0]) + params[1]) - fl_mef[0]
```

Comment:
any reason you opted to update the initial guess for `params[2]`? I'll note that I suggested this particular update back in #165, but you didn't think it would matter. also, I don't have `- fl_mef[0]` at the end of the version I proposed. Is that usually just 0? but the version you have here is more general (in case the 1st peak is ever *not* a blank)?

with that in mind, I still think my other suggestion proposed in #165 (initialize slope and y-intercept terms from a line fit with all the bead subpopulations, not just from the top two points. arguably more robust) is a good idea